### PR TITLE
W-11786742-WhatsNew7-14-0-CP

### PR DESCRIPTION
**Pull request by Crispy-Salesforce**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/whats-new-in-studio.adoc [hunk 1]
```diff
@@ -1,35 +1,31 @@
 = What’s New in Studio
 
-Anypoint Studio 7.13 introduces support for message flow visualization and management of DataWeave libraries through its UI.
+Anypoint Studio 7.14.0 introduces new features to enhance the users' experience making the daily use of Studio easier, faster, and more agile. 
 
-This version also improves the general user experience including new features such as the Exchange Comprehensive Publication.
+== Improved Copy-Paste Experience
 
-== Explore and Debug Shared Flows
+Now, users have the possibility to select if a suffix is added to the copy-pasted Mule Palette elements and to the flows to avoid having similar long names.
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/whats-new-in-studio.adoc]: to select if a suffix is added > to add a suffix

File: modules/ROOT/pages/whats-new-in-studio.adoc [hunk 2]
```diff
@@ -1,35 +1,31 @@
...
 
-Studio now supports message flow visualization of a Mule configuration file loaded from a dependency, which enables exploring and debugging shared flows.
+== Deploy to CloudHub 2.0
 
-image::explore-and-debug-shared-flows.png["A Mule Configuration file inside a dependency is highlighted in the Package Explorer and its flow displayed on the main panel."]
+Aligned with the latest release, Studio allows developers to deploy their applications to https://www.mulesoft.com/platform/saas/cloudhub-ipaas-cloud-based-integration[CloudHub 2.0], which offers a wide variety of shared services.
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/whats-new-in-studio.adoc]: Studio allows > Studio enables
Add **^** at the end of the URL to show the link as external: https://www.mulesoft.com/platform/saas/cloudhub-ipaas-cloud-based-integration[CloudHub 2.0^]

File: modules/ROOT/pages/whats-new-in-studio.adoc [hunk 3]
```diff
@@ -1,35 +1,31 @@
...
 
-== Easy Management of DataWeave Libraries
+== Project Creation Made Easier
 
-Users now have easier access to manage DataWeave libraries through the Studio UI.
+The on-screen information has been simplified and now users can choose the view of API implementation when creating a project from Studio preferences:
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/whats-new-in-studio.adoc]: can choose the view > can customize the view (please apply this suggestion only if it is accurate and reflects the actual the change)

File: modules/ROOT/pages/whats-new-in-studio.adoc [hunk 4]
```diff
@@ -1,35 +1,31 @@
...
-The Exchange Comprehensive Publication features enhance the user experience in Studio when publishing assets to Exchange.
+image::api-implementation-collapsed.png["*API Implementation* section collapsed in the *New Mule Project* window."]
 
-* Custom fields are now available to add from the wizard.
-* Users can now create and edit Markdown files and deploy them along with the asset.
-* An icon can be added to new assets when publishing them to Exchange.
+== Increased Visibility and Readability
 
-== Improved General User Experience
+In line up with accessibility initiatives, the color contrast in Studio is enhanced to improve its visibility and readability.
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/whats-new-in-studio.adoc]: _In line up with_ accessibility initiatives, the color contrast _in Studio is enhanced_ to improve its visibility and readability. >
**Following** accessibility initiatives, **Studio has an enhanced** color contrast to improve visibility and readability.

File: modules/ROOT/pages/whats-new-in-studio.adoc [hunk 5]
```diff
@@ -1,35 +1,31 @@
...
 
-Studio provides a better user experience by showing clearer information on the screen.
+== Better Performance When Creating and Opening Projects
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/whats-new-in-studio.adoc:26]: A general suggestion about titles to make them consistent by starting with a verb: 

== Improved Copy-Paste Experience > OK
== Deploy to CloudHub 2.0 > **Added** Support for CloudHub 2.0 Deployment (or) **Added** Deployment to CloudHub 2.0
== Project Creation Made Easier > **Simplified/Improved** Project Creation
== Increased Visibility and Readability > OK
== Better Performance When Creating and Opening Projects > **Improved** Performance When Creating and Opening Projects

**Review by Cristian-Venticinque — DISMISSED** [review]
Please address comments before merging

File: modules/ROOT/pages/whats-new-in-studio.adoc (hunk 1 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/whats-new-in-studio.adoc]: Merging Diane's suggestion. Thanks!

File: modules/ROOT/pages/whats-new-in-studio.adoc (hunk 3 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/whats-new-in-studio.adoc]: No customization. They can only choose to view -or not- the API Implementation section.

File: modules/ROOT/pages/whats-new-in-studio.adoc (hunk 4 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/whats-new-in-studio.adoc]: Thanks!

File: modules/ROOT/pages/whats-new-in-studio.adoc (hunk 5 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/whats-new-in-studio.adoc:26]: Thanks for the tip!
I used noun phrases and I didn't think about starting them with verbs.

File: modules/ROOT/pages/whats-new-in-studio.adoc (hunk 2 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/whats-new-in-studio.adoc]: Thanks for the tip!

**Review by juarez-fabian — APPROVED** [review]

**Review by JennyHajee — APPROVED** [review]

**Pull request merged**